krypto/src/commonMain/kotlin/com/soywiz/krypto/encoding/Base64.kt

```diff
@@ -49,31 +56,36 @@ object Base64 {
     }
 
     @Suppress("UNUSED_CHANGED_VALUE")
-    fun encode(src: ByteArray): String {
+    fun encode(src: ByteArray, isUrlSafe: Boolean = false): String {
+        val table = if (isUrlSafe) {
+            TABLE_SAFE
+        } else {
+            TABLE
+        }
```

Comment:
Looking at this. What would you think about renaming TABLE and TABLE_SAFE to: `ENCODE_TABLE` and `ENCODE_TABLE_URLSAFE` to make it clearer?

Reply:
Done